frontend/src/features/public-form/components/FormPaymentPage/components/types.ts

```diff
@@ -0,0 +1,16 @@
+import { FormColorTheme } from '~shared/types'
+
+import { PaymentItemNameDescriptionProps } from './PaymentItemNameDescription'
```

Comment:
Should payment item name props just be collapsed into this type file? Then this file will serve as the types/props file for our payment item components

Reply:
Agreed. 700d1fc6481c4055e65f081f4d1f2bfb041ce7ed